main.py

```diff
@@ -1,7 +1,40 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    for genre_name in ("Western", "Action", "Dramma"):
```

Comment:
The genre name 'Dramma' seems to be a typo. It is later corrected to 'Drama', but it's better to use the correct name from the start to avoid unnecessary updates.